canonical/store-components
# Create initial footer component

## Done
Created initial `footer` component

## QA
- Go to https://store-components-14.demos.haus/?path=/story/footer--page
- Check that there is a footer.

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-1613
